disco_aws_automation/disco_bake.py

```diff
@@ -369,7 +369,7 @@ def bake_ami(self, hostclass, no_destroy, source_ami_id=None, stage=None):
         reservation = self.connection.run_instances(
             source_ami_id,
             block_device_map=self.disco_storage.configure_storage(
-                hostclass, ami_id=source_ami_id, map_snapshot=False),
+                hostclass, ami_id=source_ami_id),
```

Comment:
'map_snapshot' was used to signal whether to create the mapping for EBS volume snapshot. Since we no longer support EBS volume here, it's no longer needed.